db/models.py

```diff
@@ -46,3 +56,63 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class User(AbstractUser):
+    pass
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,
+                             on_delete=models.CASCADE)
+
+    def __str__(self) -> str:
+        return f"{str(self.created_at)}"
+
+    class Meta:
+        ordering = ["-created_at"]
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(MovieSession, on_delete=models.CASCADE)
+    order = models.ForeignKey(Order, on_delete=models.CASCADE)
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    class Meta:
+        constraints = [
+            UniqueConstraint(fields=["movie_session", "row", "seat"],
+                             name="unique_seat")
+        ]
+
+    def __str__(self) -> str:
+        return f"{self.movie_session.movie.title} " \
+               f"{str(self.movie_session.show_time)} " \
+               f"(row: {self.row}, seat: {self.seat})"
```

Comment:
Try to avoid '\'. Use ( for this
```suggestion
        return (
            f"{self.movie_session.movie.title} "
            f"{str(self.movie_session.show_time)} "
            f"(row: {self.row}, seat: {self.seat})"
       )
```